gpu/people/tools/people_tracking.cpp

```diff
@@ -125,5 +125,6 @@ int main(int argc, char** argv)
   }
   /// Run the app
   app.run();
+  delete(app.m_proc);
```

Comment:
This is not the way to go.  `app.m_proc` is part of `PeopleTrackingApp` an is should be released if needed by its destructor.

Furthermore, the tree loading code from lines 114-125 should be encapsulated in a single method.

Reply:
I have made the changes in 6c1e94b